frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -41,6 +49,8 @@ export const breadcrumbsForVMPage = (match: any) => () => [
 
 export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProps> = (props) => {
   const { name, ns: namespace } = props.match.params;
+  const [snapData, snapLoaded, snapErr] = useK8sGet(VirtualMachineSnapshotModel);
+  const isSnapshotSupported = snapData && snapLoaded && !snapErr;
```

Comment:
can you please look at the original description of this bug https://bugzilla.redhat.com/show_bug.cgi?id=1881125 ? And fix the loading when reintroducing the snapshots again?